src/settings.rs

```diff
@@ -170,7 +170,7 @@ impl Settings {
             Ok(val) => {
                 s.set("server.port", val).unwrap();
             }
-            Err(e) => warn!("couldn't interpret PORT: {}", e),
+            _ => (),
```

Comment:
Rather than completely removing the warning message, can you move it the `Ok()` block and convert it into an `info` message?

I've faced issues trying to host other software where I couldn't tell if my environment variable was being read or not :sweat_smile:

Reply:
```suggestion
If let Ok(val) = env::var("PORT") {
   s.set("server.port", val).unwrap();
   log:info!("Overriding port with environment variable");
}
 
 ```

Reply:
I have added a few more points of logging in this file: https://github.com/mCaptcha/mCaptcha/pull/46/commits/9fd8ffd66644a03fcc49357ef1e6e26cf052a855